tools/src/main/java/org/apache/kafka/tools/PushHttpMetricsReporter.java

```diff
@@ -316,4 +316,17 @@ public Object value() {
             return value;
         }
     }
+
+    // The signature for getInt changed from returning int to Integer so to remain compatible with 0.8.2.2 jars
```

Comment:
When are we going to drop support for 0.8.x? :)

Reply:
Heh, I was actually thinking about that too since we could just remove the affected tests. It was only affecting `kafkatest.sanity_checks.test_performance_services.PerformanceServiceTest` which doesn't seem that critical anyway. But since I was able to work out how to make it compatible I figured I'd leave it for now. More generally the major thing I think we evaluate for such old versions is compatibility. It would probably be a good idea to collectively agree on a reasonable range of versions (or based on time) so we can cap how far back we need to continue maintaining tests. 2 years might be a good range, in which case we'd be able to drop tests for the 0.9 line soon too.